src/olympia/scanners/tasks.py

```diff
@@ -438,13 +440,25 @@ def call_mad_api(all_results, upload_pk):
             return results
 
         with statsd.timer('devhub.mad'):
-            json_payload = {'scanners': {'customs': customs_results.results}}
-            response = requests.post(
-                url=settings.MAD_API_URL,
-                json=json_payload,
-                timeout=settings.MAD_API_TIMEOUT,
-                headers={'x-request-id': request_id},
-            )
+            with requests.Session() as http:
+                adapter = HTTPAdapter(
+                    max_retries=Retry(
+                        total=1,
```

Comment:
Does this mean we're retrying once, and then giving up?

Reply:
yes